package.json

```diff
@@ -14,6 +14,11 @@
   "dependencies": {
     "@ethersproject/providers": "^5.6.4",
     "@ethersproject/wallet": "^5.6.2",
+    "@ethersproject/keccak256": "^5.6.1",
+    "@ethersproject/abi": "^5.6.1",
+    "@ethereumjs/block": "^3.6.3",
+    "@ethereumjs/common": "^2.6.5",
```

Comment:
Are you sure we can't get these functions from Ethers.js directly?

Reply:
ethers.js doesnt provide the functionality to encode raw block data unfortunately - which is what we need for handling storage proofs. 

These utilities are only needed during proposal creation and not for every vote so we should be able to keep the voting logic lighter.